pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -180,6 +181,17 @@ func decompressData(descBytes []byte) ([]byte, error) {
 	return ioutil.ReadAll(r)
 }
 
+// ReadBackupManifest is the export version of readBackupManifest,
+// used in cliccl backup inspection.
+func ReadBackupManifest(
```

Comment:
I think that using `readBackupManifestFromStore` (and exporting it) would be what we want here. Old backup manifests may have a different file name, and that method checks handling both.